models/property.py

```diff
@@ -47,6 +47,12 @@ def json(self):
             "updated_at": Time.format_date(self.updated_at),
         }
 
+    def tenants(self):
+        tenants = []
+        for lease in self.leases:
+            tenants.append(lease.tenant.json())
+        return tenants
```

Comment:
I would really like to know if sql alchemy offers a way to declare that a property has many tenants through the lease relationship. If not, it would probably be fun to create our own dsl to declare those types of relationships On the model.